### ISSUE TITLE
§III.17(4) discriminatory taxes

### ISSUE THREAD

**Issue by yopdog** [liberland/constitution#323]
If we end up having taxes, they should at least be equal for everyone. I'm therefore very against the following part that was add: 

>§III.17(4) The fee shall be payable once a year by **some** or all owners of land within the territory of the Free Republic of Liberland other than the Public Administration, as prescribed in the Bill.   

I think it's dangerous to have a tax that isn't equal for everyone. It's part of the process that corrupts the public in every democracy. People would always try and get others to pay the burden of taxes while excluding themselves. Elections will be based on who can get the rich minority to pay most of the taxes. Do we really want this type of thing happening in liberland? If we have to have taxes it must at the very least be fair. Meaning the SAME for everyone.

**Comment by Auahi** [comment]
It looks scary I know but it's designed so the assembly could tax business and large corporations only, that way the only people who have to pay taxes are paying making big money... and again the tax bills have to be approved by the citizens so this legal system is very different to that of other countries.

**Comment by Auahi** [comment]
It's designed so the assembly could tax commercial buildings only. Also the tax bills have to be approved by the citizens so this legal system is very different than that of other countries. So if an unfair tax came out it would be immediately rejected.

**Comment by yopdog** [comment]
>It's designed so the assembly could tax commercial buildings only

That's exactly what I don't like about it. It taxes based on wealth instead of having everyone pay the same. Furthermore, How will you determine the difference between commercial property and residential? What if you have something in between? This would just make the tax system more complicated and expensive to run.

**Comment by Auahi** [comment]
It's not based on wealth it's based on type of business done on a property. 

Taxing residential buildings is completely different't, like I said in a post in another issue, property tax completely ignores the personal aspect of a property, lets say property taxes are 16500 for the average lot, that 16500 is a completely different number person to person. To someone who makes 100000 it's not a lot but to someone who makes 30000 it's more than half their paycheck, so you can't call taxing everybody through property taxes is fair.

This won't make the tax system more complicated the assembly can just make a tax bill stating "Property tax will be assessed on large commercial properties only(Aka skyscrapers big grocery stores). The assembly could provide a pardon to small business owners such as mom and pops shops. It's not complicated at all.

**Comment by Auahi** [comment]
the only way I could see taxing everyone through property taxes to be fair would be the maximum amount you pay will be respective to wage for example: The amount of yearly property taxes can not exceed 5% of one's annual wage.

**Comment by yopdog** [comment]
@Auahi the whole point of this land tax is that it NOT based on income. Think of this land tax as a fee (which is what it's actually called) that you pay to live within the territory of Liberland. The more land you own the bigger the fee you pay. the size of the fee should in no way be connected to what you earn or the business you run on your property. Saying that those who have more money should pay more is what's wrong with the world and is what Liberland is going to fix.

**Comment by Auahi** [comment]
I'm not saying those with more money have to pay more, that is RIDICULOUS, what I'm saying is land tax can cripple a persons pay check! two people who own the same property size lets say the taxes are 10000 one person makes 200 000 and the other makes 300 000, they will pay the same amount since it is not over 5% of their annual income, but a person who makes 30 000 would not be able to afford 10000 in taxes, that's whats wrong with land taxes, it disregards people's availability to pay!

**Comment by yopdog** [comment]
Then the land tax will have to be low enough to allow the vast majority of working people to pay them. It basically comes down to keeping taxes low.

**Comment by Auahi** [comment]
Okay that I can agree it's just that with how the tax system is setup now it could get high fast.

**Comment by yopdog** [comment]
Ok great, that's a good topic for another discussion. As for this part, do you agree it should be removed?

**Comment by Auahi** [comment]
I'm reluctant to say yes, maybe we should get more opinions on this but I'm in the middle right now

**Comment by Auahi** [comment]
The only other point wrong I see is people over 70 they have a limited income already, that would have been the other group I would have excluted

**Comment by Auahi** [comment]
@KacperZajc @terrorist96 you're thoughts?

**Comment by KacperZajc** [comment]
Both points are valid.
Im happy either way.
@terrorist96 ???

**Comment by terrorist96** [comment]
If a millionaire has the same amount of land as a janitor, either the millionaire is being very cheap (to avoid the land fee) or the janitor has bitten off more than he can chew and should decrease his land ownership to a more reasonable level.
I agree with the fine line between residential and commercial. What would renting a room out in your house on Airbnb count as?

Another thought: a skyscraper and a residence could take up the same amount of land. The skyscraper is just more efficient because it uses 'land area' in the air (by building upwards, not outwards). The tax should be based on total square footage, not land in general.

Here's my suggestion:


> §III.17(3) The fee shall be fixed per meter square of ~~land~~ **area**.
§III.17(4) The fee shall be payable once a year by ~~some or all owners of land~~ **landowners** ~~within the territory of the Free Republic of Liberland~~**,** other than the Public Administration, as prescribed in the Bill.

**Comment by KacperZajc** [comment]
Im not sure this is a good idea:

1) Taxing actual land would ensure efficiency thereby constructing taller buildings (which is a good thing regarding that we have only 7km2 of land)
2) It is easier to keep record of the size of actual land rather then inspecting the total square footage of every building erected. 

Im not saying it's a bad idea.

**Comment by terrorist96** [comment]
Yeah, I'm not totally committed to my proposal either. I'm just shooting out ideas.

**Comment by KacperZajc** [comment]
let's just go with your idea of 'landowners' (and not with 'area'), it seems good to me

**Comment by terrorist96** [comment]
:+1:

**Comment by Auahi** [comment]
Terrorist many people own land the same size but the house value is what matters. Liver land is small so people will build up to out, so it's perfectly feasable for two people to have the same land
